src/ethereum/testnets/__init__.py

```diff
@@ -0,0 +1,16 @@
+"""
+Testnet Configuration
+^^^^^^^^^^^^^^^^^^^^^
+
+.. contents:: Table of Contents
+    :backlinks: none
+    :local:
+
+Introduction
+------------
+
+This module contains code and configuration information enabling the
+specification to be used with Ethereum testnets.
+
+This module does not form part of the Ethereum specification.
```

Comment:
Would this make sense as a separate module, like `src/ethereum_testnets/...` or is that not possible?